src/common/subscription/SubscriptionViewer.ts

```diff
@@ -381,6 +394,61 @@ export class SubscriptionViewer implements UpdatableSettingsViewer {
 		}
 	}
 
+	private async canManageAppStoreSubscriptionInApp(accountingInfo: AccountingInfo, ownership: MobilePaymentSubscriptionOwnership): Promise<boolean> {
+		if (!accountingInfo.appStoreSubscription) {
+			throw Error("Trying to manage an non-existing app store subscription")
```

Comment:
what errors are these? Are they `ProgrammingError`s?

Reply:
This specific one can be removed, we already check the presence of appstoreSubscription on line 328.
The other one below on the same method I'd say yes, that's a ProgrammingError